catalog/app/components/TalkToUs/TalkToUs.js

```diff
@@ -72,16 +72,19 @@ export function TalkToUsProvider({ children }) {
         w.show()
       })
     },
-    [calendlyP, t.track],
+    [t, cfg.calendlyLink, calendlyP],
   )
 
   return <Ctx.Provider value={showPopup}>{children}</Ctx.Provider>
 }
 
 export function useTalkToUs(extra) {
-  const show = React.useContext(Ctx)
-  const bound = React.useCallback(() => show(extra), [show])
-  return extra ? bound : show
+  const ref = React.useRef({
+    bound: (localExtra) => ref.current.show(ref.current.extra || localExtra),
+  })
+  ref.current.show = React.useContext(Ctx)
+  ref.current.extra = extra
+  return ref.current.bound
```

Comment:
Where can I find this page?

Reply:
on the landing page of the website (mode === 'MARKETING') in the following places:
- `app/website/pages/Landing/Pricing`: pricing grid "talk to us" and "contact" btns
- `app/website/pages/Landing/Showcase`: hero, "book a demo" btn
- `app/website/pages/Landing/StickyFooter`: sticky footer "talk to us" btn

Reply:
(it's not a page, just a function opening the calendar popup)

Reply:
Is it deployed somewhere already?

Reply:
> it's not a page
dude

Reply:
> Is it deployed somewhere already?

nope, but you can set `"mode": "MARKETING"` in config.json to test it locally

Reply:
If it isn't deployed to production, it looks like dead code for me. I don't want to test dead code :)

Reply:
oh, i see what you mean, it's deployed on https://quiltdata.com. i thought you were asking is this particular revision deployed somewhere